docs/io/optional/logging_configuration.ipynb

```diff
@@ -0,0 +1,2226 @@
+{
```

Comment:
<p>Reword first sentence to be "A new message can be seen from the execution of the simulation,"</p><p>...</p><p>"that is informing the user which input log level...".</p><p>Removal trailing space after output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>